api/graphql/schema.graphql

```diff
@@ -1225,4 +1267,17 @@ type Mutation {
     deleteCommunityExperience(id: ID!): CommunityExperience @delete(globalId: false) @guard @can(ability: "delete", find: "id")
     deleteEducationExperience(id: ID!): EducationExperience @delete(globalId: false) @guard @can(ability: "delete", find: "id")
     deleteAwardExperience(id: ID!): AwardExperience @delete(globalId: false) @guard @can(ability: "delete", find: "id")
+
+    # PoolAdvertisement mutations
+    createPoolAdvertisement(userId: ID! @rename(attribute: "user_id"), poolAdvertisement: CreatePoolAdvertisementInput! @spread): PoolAdvertisement @create(model: "Pool") @guard @can(ability: "create", model: "Pool")
```

Comment:
I'm unsure of the use-cases for some of these mutations. For example:
When would we prefer this create mutation vs the createPool mutation when they do basically the same thing?
And why would we prefer each of these specific update mutations vs a more general one?

Reply:
My understanding was to make explicit mutations for the new Pool Advertisement model and the events on the create and edit pages. However, you do have a point since a Pool Advertisement is just a Pool. @tristan-orourke Thoughts on this?